src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -232,6 +232,24 @@ module.exports = class LevelBlock {
   }
 
   /**
+   * Can this block be placed in liquid to replace a liquid block? Should
+   * generally be true for all "standard" blocks like cobblestone and dirt, and
+   * false for all "special" blocks like redstone and torches.
+   *
+   * @return {boolean}
+   */
+  getIsPlaceableInLiquid() {
```

Comment:
I wonder if we want to generalize the idea of a normal "block" vs. non-blocks.  Whether not something is a block has implications about whether or not it's walkable, etc.  I _think_ all walkable blocks can't be placed in water?

But this is great first-pass cleanup and bug-fix!